msu/hooks/skills/skill.nut

```diff
@@ -4,10 +4,11 @@
 
 	q.create = @(__original) function()
 	{
-		__original();
 		if (this.m.DamageType == null)
 			this.m.DamageType = ::MSU.Class.WeightedContainer();
 
+		__original();
+
```

Comment:
Is this order change intended?

Reply:
Yes that's part of a separate fix commit. It makes sense to me that we initialize it before calling the original function otherwise someone may hook the original function and try to interact with the DamageType field but it's null at that time.